<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::Element</class>
 <widget class="QWidget" name="Sekura::Element">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>LineEdit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DateTime</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ComboBox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SpinBox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DataType</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ForeignKey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Element</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox1"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox2"/>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox2"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="5"/>
 <resources/>
 <connections/>
</ui>
